contributing/videos.md

```diff
@@ -0,0 +1,18 @@
+(videos)=
+# Contribute timestamps to videos
+
+## Requirements
+- GitHub account
+- Markdown
+
+## About: why we are adding timestamps
+
+We have a collection of videos on YouTube from PyMCon 2020 and PyMCon Web Series.
+
+:stopwatch: We are adding timestamps to the PyMC YouTube videos.  When timestamps are available:
```

Comment:
This doesn't render. I think copying the emoji directly instead of its id will work, otherwise use one of https://sphinx-design.readthedocs.io/en/sbt-theme/badges_buttons.html#inline-icons